docs/reference/analysis/analyzers/stop-analyzer.asciidoc

```diff
@@ -110,14 +116,36 @@ The `stop` analyzer accepts the following parameters:
 [horizontal]
 `stopwords`::
 
-    A pre-defined stop words list like `_english_` or an array  containing a
+    A pre-defined stop words list like `_english_` or an array containing a
     list of stop words.  Defaults to `_english_`.
 
 `stopwords_path`::
 
     The path to a file containing stop words. This path is relative to the
     Elasticsearch `config` directory.
 
+The default English stopwords used in Elasticsearch are:
+
+    a, an, and, are, as, at, be, but, by, for, if, in, into, is, it,
+    no, not, of, on, or, such, that, the, their, then, there, these,
+    they, this, to, was, will, with
+
+
+==== Stop words in supported languages
+
+To set the default stopwords list for a particular language, use the
+_lang_ notation:
+
+[source,text]
+---------------------------------
+"stopwords": "_french_"
+---------------------------------
+
+Each language analyzer defaults to using the appropriate stopwords list
+for that language.
+
+Stop words in other supported languages can be accessed in Lucene:
+https://github.com/apache/lucene-solr/tree/master/lucene/analysis/common/src/resources/org/apache/lucene/analysis[Stopwords]
```

Comment:
It might be worth mentioning that some of these are under the `snowball` path:
https://github.com/apache/lucene-solr/tree/master/lucene/analysis/common/src/resources/org/apache/lucene/analysis/snowball

Reply:
The list of stopwords for each language is different from snowball. You can find the entire list here:
https://github.com/elastic/elasticsearch/blob/master/server/src/main/java/org/elasticsearch/index/analysis/Analysis.java#L115